src/guest/vm_flash.cc

```diff
@@ -172,21 +172,22 @@ bool VmFlasher::FlashWithQemu(void) {
 
     std::string rpmb_bin = cfg_.GetValue(kGroupRpmb, kRpmbBinPath);
     std::string rpmb_data_dir = cfg_.GetValue(kGroupRpmb, kRpmbDataDir);
-    std::string rpmb_data_file = rpmb_data_dir + "/" + std::string(kRpmbSock);
+    time_t rawtime;
+    struct tm timeinfo;
+    char t_buf[80];
+    time(&rawtime);
+    localtime_r(&rawtime, &timeinfo);
+    strftime(t_buf, 80 , "%Y-%m-%d_%T", &timeinfo);
+    std::string rpmb_sock = std::string(rpmbSockDir) + t_buf;
+
     std::unique_ptr<VmCoProcRpmb> rpmb_proc;
     boost::system::error_code ec;
     if (!rpmb_bin.empty() && !rpmb_data_dir.empty()) {
-        if (boost::filesystem::exists(rpmb_data_file))  {
```

Comment:
Add logic to remove rpmb_data_dir/kRpmbData. The RPMB_DATA need to be regenerated when flash a new CiV.